src/main/java/frc/trigon/robot/constants/AutonomousConstants.java

```diff
@@ -1,15 +1,47 @@
 package frc.trigon.robot.constants;
 
+import com.pathplanner.lib.auto.NamedCommands;
 import com.pathplanner.lib.path.PathConstraints;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 
 public class AutonomousConstants {
     public static final PathConstraints REAL_TIME_CONSTRAINTS = new PathConstraints(2.5, 2.5, 4, 4);
+    public static boolean shouldUpdateRobotAngle = false;
 
     static {
         registerCommands();
     }
 
     private static void registerCommands() {
-//        NamedCommands.registerCommand(name, command);
+        NamedCommands.registerCommand("Collect", new InstantCommand(
+                () -> {
+                    System.out.println("Collecting");
+                    shouldUpdateRobotAngle = false;
+                }));
+        NamedCommands.registerCommand("WarmShooting", new InstantCommand(
+                () -> {
+                    System.out.println("WarmingShooting");
+                    shouldUpdateRobotAngle = false;
+                }));
+        NamedCommands.registerCommand("PrepareForShooting", new InstantCommand(
+                () -> {
+                    System.out.println("PreparingForShooting");
+                    shouldUpdateRobotAngle = true;
+                }));
+        NamedCommands.registerCommand("FeedNote", new InstantCommand(
+                () -> {
+                    System.out.println("FeedingNote");
+                    shouldUpdateRobotAngle = true;
```

Comment:
We expect it to either already be at the angle when feeding, or we don't want to do it automatically